appEntry.js

```diff
@@ -0,0 +1,20 @@
+import { registerRootComponent } from "expo";
+import { ExpoRoot } from "expo-router";
+import { setPushConfig } from "./utils/setPushConfig";
+import { setNotifeeListeners } from "./utils/setNotifeeListeners";
+import { setFirebaseListeners } from "./utils/setFirebaseListeners";
+import { setPushMessageHandlers } from "./utils/setPushMessageHandlers";
+
+setNotifeeListeners();
+setFirebaseListeners();
+setPushMessageHandlers();
+setPushConfig();
+export function appEntry() {
+  const ctx = require.context("./app");
```

Comment:
this seems unusual. is there any documentation showing something like this?

Reply:
changed app entry to allow headless activity
similar to below issue, we required to allow few config to run in kill mode
https://github.com/transistorsoft/react-native-background-fetch/issues/493